deployment-examples/aws/extension/README.md

```diff
@@ -0,0 +1,13 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  
```

Comment:
I found TOCs valuable on when the things are complex and needs navigation, in this document it's quite easy to see what is this about, is not it ?

Reply:
41f888c3f1faf527e3e798aec08984fa7077d7e7